### PR TITLE
Css canvas medical - Chivy Chia

### PR DESCRIPTION
**Pull request by tinachia**
image

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+          <div class="column-half">
+          </div>
+          <div class="column-half">
+            <div>
+              <button class="button-bottom">For Health Plan</button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half">
```
Comment (Zhiwutian) [inline css-canvas-medical/index.html:213]: make sure to remove empty elements like this, if you are using it for spacing, I would recommend going for the same result using flexbox

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking really good Tina! Just make sure to remove any elements which don't have content

**Comment by tinachia** [comment]
image
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**